docs/advanced/input_files/input-main.md

```diff
@@ -1015,6 +1016,17 @@ We recommend the following options:
 - **Description**: the minimum kerker coefficient 
 - **Default**: 0.1
 
+### mixing_angle
```

Comment:
Please update the description in https://abacus.deepmodeling.com/en/latest/advanced/scf/converge.html#charge-mixing for new mixing parameters.